sdcm/cluster.py

```diff
@@ -4717,6 +4717,19 @@ def node_setup(self, node: BaseNode, verbose: bool = False, timeout: int = 3600)
         node.clean_scylla_data()
         node.remoter.sudo(cmd="rm -f /etc/scylla/ami_disabled", ignore_status=True)
 
+        if append_scylla_node_exporter_args := self.params.get('append_scylla_node_exporter_args'):
+            configuration_file = '/etc/default/scylla-node-exporter'
+            with remote_file(remoter=node.remoter,
+                             remote_path=configuration_file,
+                             serializer=properties.serialize,
+                             deserializer=properties.deserialize,
+                             sudo=True) as properties_file:
+                current_args = properties_file.get('SCYLLA_NODE_EXPORTER_ARGS', '')
+                if append_scylla_node_exporter_args not in current_args:
```

Comment:
basically it will be working ok for single arg, if multiple are provided then this will probably always resolve to True.
If `append_scylla_node_exporter_args` was a list then we could iterate each arg or split it to verify one by one.

Reply:
for now it's good enough for what's needed